plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ModuleWrappingTransformation.xtend

```diff
@@ -283,6 +284,10 @@ class ModuleWrappingTransformation extends Transformation {
 					completeModuleSpecifier
 				};
 
+				if (isReactImport && !actualModuleSpecifier.startsWith('@@cjs')) {
```

Comment:
Don't we have constant for this somewhere?

Reply:
I have refactored the code. This string goes away.